examples/pingone-neo-getting-started/versions.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
```

Comment:
The linter is broken, but would pick up on a missing line needed between lines 1 and 2:
`required_version = ">= 1.1.0"`